Task_Position_the_Cubes/index.css

```diff
@@ -1 +1,54 @@
-/*  Start coding below for POSITION THE CUBES TASK*/
+/*  Start coding below for POSITION THE CUBES TASK*/
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.parent{
+    height: 300px;
+    width:300px;
+    position: relative;
+    background-color: #675BC6;
+}
+
+.child{
+    position: absolute;
+    height: 100px;
+    width: 100px;
+}
+
+.one{
+    border: 2px solid white;
+    background-color: #60B177;
+}
+
+.two{
+    left: 200px;
+    right: 200px;

```

Comment:
The properties 'left' and 'right' should not be used together in the '.two' class. Choose either 'left' or 'right' to position the element horizontally, not both.